src/main/java/racing/ParserUtils.java

```diff
@@ -0,0 +1,29 @@
+package racing;
+
+import java.util.Optional;
+
+public class ParserUtils {
+
+    private static final String COMMA = ",";
+
+    public static String[] stringToArray(String input) {
+        return Optional.ofNullable(input)
+                .filter(text -> !text.trim().isEmpty())
+                .map(text -> text.split(COMMA))
+                .orElseThrow(IllegalArgumentException::new);
+    }
+
+    public static void checkEmptyOrBlank(String[] nameList) {
+        for (String name : nameList) {
+            isEmpty(name);
+        }
+    }
+
+    private static void isEmpty(String inputName) {
+        if (inputName.trim().isEmpty()) {
+            throw new IllegalArgumentException("이름이 입력되지 않았습니다.");
+        }
+    }
```

Comment:
아래처럼 변경해 보면 어떨까요?
아래의 구조로 만들면, isBlank() 메소드를 stringToArray() 메소드의 12 번째 줄에서도 사용가능해 보여요.
```suggestion
    private static void checkBlank(String inputName) {
        if (isBlank(inputName)) {
            throw new IllegalArgumentException("이름이 입력되지 않았습니다.");
        }
    }
    
    private static boolean isBlank(String str) {
        return str == null || str.trim().isEmpty();
    }
```

Reply:
완료